crates/accelerate/src/target_transpiler/qargs.rs

```diff
@@ -132,6 +132,16 @@ impl QargsRef<'_> {
     pub fn is_concrete(&self) -> bool {
         !self.is_global()
     }
+
+    /// Converts into an owned version of [Qargs]
+    pub fn as_owned_qargs(&self) -> Qargs {
+        match self {
```

Comment:
Should this just be the `ToOwned` trait or something similar. The method name here `as_owned_qargs` is a bit weird looking, especially when considering: https://rust-lang.github.io/api-guidelines/naming.html#ad-hoc-conversions-follow-as_-to_-into_-conventions-c-conv it would normally be called `to_owned()`

That being said isn't this just used for error messages, it feels like maybe we should adjust or loosen the typing on the error type arguments so we can just call debug on the `QargsRef` instead of requiring an owned copy.

Reply:
I agree on us improving the typings on the errors here to avoid the need for this method, since that's the only reason we use it. 

I wanted to have the method named `to_owned` but didn't want to imply that this implements `ToOwned<Owned = Qargs>`mainly because we're unable to implement `Borrow<QargsRef>` for `Qargs`. That said, I'm more than happy to change/remove the method.

Reply:
Removed in 4ae2e71